uwblueprint/paramedics-react
# Fix events not being shown upon creation

[Notion ticket](https://www.notion.so/uwblueprintexecs/396370c5e09f4c28bc7f455a7e84c908?v=3751aa62a2594c74b7e0ecb363304c24&p=51b9ef15e75c49508a18eb5ea8e52795)

## Changes
- Added back event query with `@client` directive, `GET_EVENTS_FROM_CACHE`
- Changed where the `history.replace` was happening after creating an event to fix async issues
- Renamed `useAllEvents` to `useAllEventsFromBackend`

## Screenshots
N/A

## Testing
- Add multiple new event, and ensure that the events are shown in the landing page highlighted in blue
- Ensure that the created event is the last event card
- Edit an event, and see if changes are reflected immediately
- Archive an event
- Delete an event

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
